lib/zendesk_apps_support/validations/requests.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'uri'
+require 'ipaddress'
+
+module ZendeskAppsSupport
+  module Validations
+    module Requests
+      class << self
+        HTTP_REQUEST_METHODS = %w[GET HEAD POST PUT DELETE CONNECT OPTIONS TRACE].freeze
+
+        REQUEST_CALLS = [
+          /\w+\.request\(['"](.*)['"]/i, # ZAF request
+          /\$\.(?:get|post|getJSON)\(['"](.*)['"]/i, # jQuery request
```

Comment:
Should we also check for the`ajax` function?
Also worth checking for `jQuery.{function_name}` as well as `$.{function_name}`?

Reply:
Mm yep, good call. I'll update these